### ISSUE TITLE
Idea: Allow dragging an image into window

### ISSUE THREAD

**Issue by ejdub** [Savjee/bearclaw#12]
It would be nice to be able to drag screenshots from the desktop or other application into the Bearclaw window, rather than having to use the built in screenshot tool.

**Comment by Savjee** [comment]
True! I'll see what I can do for that. Not really an expert in Swift development so if you know someone, tag them ;)
